docs/README.md

```diff
@@ -0,0 +1,42 @@
+Documentation
+==================
```

Comment:
Everything looks good. One nit: please make underlines as long as the titles, not shorter and not longer.

Reply:
Thank you, I didn't notice